conn/bind_std.go

```diff
@@ -6,64 +6,133 @@
 package conn
 
 import (
+	"context"
 	"errors"
 	"net"
 	"net/netip"
+	"strconv"
 	"sync"
 	"syscall"
+
+	"golang.org/x/net/ipv4"
+	"golang.org/x/net/ipv6"
+)
+
+var (
+	_ Bind = (*StdNetBind)(nil)
 )
 
 // StdNetBind is meant to be a temporary solution on platforms for which
 // the sticky socket / source caching behavior has not yet been implemented.
```

Comment:
I noticed bind_linux.go was deleted in this PR. Intentional? bind_std.go is intended to replace it? If so, this comment could be updated I suppose?